nodl_python/test/nodl.xml

```diff
@@ -6,7 +6,7 @@
 
   <node name="node_2" >
     <parameter name="rate" type="int" />
-    <topic name="/foo/bar" type="std_msgs/msg/String" subscription="true">
+    <topic name="/foo/bar" type="std_msgs/msg/String" subscriber="true">
```

Comment:
In ROS2 wording, 'subscriber' has become 'subscription', see e.g. [here](https://github.com/ros2/examples/blob/master/rclcpp/minimal_subscriber/lambda.cpp#L36).

Reply:
Fixed